pkg/util/mount/mount.go

```diff
@@ -175,27 +174,26 @@ func GetDeviceNameFromMount(mounter Interface, mountPath string) (string, int, e
 // getDeviceNameFromMount find the device name from /proc/mounts in which
 // the mount path reference should match the given plugin directory. In case no mount path reference
 // matches, returns the volume name taken from its given mountPath
-func getDeviceNameFromMount(mounter Interface, mountPath, pluginDir string) (string, error) {
-	refs, err := GetMountRefs(mounter, mountPath)
+func getDeviceNameFromMount(logger *zap.SugaredLogger, mounter Interface, mountPath, pluginDir string) (string, error) {
+	refs, err := GetMountRefs(logger, mounter, mountPath)
```

Comment:
Rather than doing `With("mountPath", mountPath)` multiple times (note no spaces in field names) I'd instead reassign logger:

```go
logger := logger.With("mountPath", mountPath)
```